app/components/works/contributor_row_component.html.erb

```diff
@@ -63,6 +63,10 @@
             </div>
           </div>
         </fieldset>
+        <%# This can be removed after affiliation implementation. %>
+        <% if Settings.ror_demo %>
+          <%= render("works/organization_input") %>
+        <% end %>
```

Comment:
If we didn't want this logic living in a view, we could make the `works/organization_input` view a new view component, and the Ruby side of the component could have:

```ruby
def render?
  Settings.ror_demo
end
```

But since this code looks like it's going to be remove before too long, it may not be worth the effort?

Reply:
Yeah, the input is throw away code.